Ports/libzip/patches/disable_unnecessary_options.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 017f7cf0..125900e2 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -9,19 +9,23 @@ project(libzip
+   VERSION 1.7.3
+   LANGUAGES C)
+ 
+-option(ENABLE_COMMONCRYPTO "Enable use of CommonCrypto" ON)
+-option(ENABLE_GNUTLS "Enable use of GnuTLS" ON)
+-option(ENABLE_MBEDTLS "Enable use of mbed TLS" ON)
+-option(ENABLE_OPENSSL "Enable use of OpenSSL" ON)
+-option(ENABLE_WINDOWS_CRYPTO "Enable use of Windows cryptography libraries" ON)
++set(ZLIB_LIBRARY /home/baitinq/serenity/Build/i686/Root/usr/local/lib/libz.a)
++set(ZLIB_INCLUDE_DIR /home/baitinq/serenity/Build/i686/Root/usr/local/include/zlib.h)
```

Comment:
that's not going to work

Reply:
oh lmao forgot about that, ill fix it in a sec